frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -3044,6 +3045,7 @@
         vm.acceptTermsAndConditions = function (acceptTermsAndConditionsForm) {
             if (acceptTermsAndConditionsForm) {
                 if (vm.termsAndConditions) {
+                    vm.warningFlag=true;
```

Comment:
@Harshit28j This approach assumes that the participant will do submission after accepting the terms. But if the page is refreshed after that then this doesn't work. Try to come up with some way to identify if he has made the submission through the API calls.